README.md

```diff
@@ -32,44 +28,26 @@ docker push eu.gcr.io/census-rm-ci/census-rm-sample-loader:<TAG>
 ```
 
 ## Testing Locally with Docker
-To test the script locally you must run RabbitMQ and Redis containers. A docker-compose.yml file exists for this purpose.
+To test the script locally you must run a RabbitMQ containers A docker-compose.yml file exists for this purpose.
 
 ```
 docker-compose up -d
 ```
 
-Once running Redis and RabbitMQ are running you can now run the sample loader.
+Once RabbitMQ is running you can now run the sample loader.
 
 ## Usage
 ```
-python load_sample.py sample.csv <COLLECTION_EXERCISE_UUID> <ACTIONPLAN_UUID> <COLLECTION_INSTRUMENT_UUID>
+python load_sample.py sample.csv <COLLECTION_EXERCISE_UUID> <ACTIONPLAN_UUID>
```

Comment:
If running locally in a virtual environment, this should probably be `pipenv run python load_sample.py sample.csv <COLLECTION_EXERCISE_UUID> <ACTIONPLAN_UUID>`